README.md

```diff
@@ -87,12 +68,73 @@ Learn from experienced cooks and grocery shoppers and provide the resulting guid
 
 ## Getting Started
 
-In this phase of the project, all raw data exist in CSV's as downloaded from Instacart [as linked below](##DataLinks). In order to get up and running yourself, you will need to download these large files and move all `.csv` files into `./data/external/`
+In this phase of the project, all raw data exist in CSV's as downloaded from Instacart [as linked below](##DataLinks). In order to get up and running yourself, you will need to download these large files into `./data/external/` along with several other setup steps required before running the application.
+
+Luckily for you, all this can be easily automated using the included Makefile
+
+### Using the Makefile
```

Comment:
I see the instructions here. Maybe move the charter somewhere else so it might not be confusing as another markdown file in the repo?